examples/prostate/pt/learners/supervised_learner.py

```diff
@@ -23,20 +23,22 @@
 from nvflare.apis.dxo import DXO, DataKind, MetaKey, from_shareable
 from nvflare.apis.fl_constant import FLContextKey, ReturnCode
 from nvflare.apis.fl_context import FLContext
-from nvflare.apis.shareable import Shareable, make_reply
+from nvflare.apis.shareable import ReservedHeaderKey, Shareable, make_reply
 from nvflare.apis.signal import Signal
 from nvflare.app_common.abstract.learner_spec import Learner
+from nvflare.app_common.abstract.model import ModelLearnableKey
 from nvflare.app_common.app_constant import AppConstants, ModelName, ValidateType
 from nvflare.app_common.pt.pt_fedproxloss import PTFedProxLoss
 
 
 class SupervisedLearner(Learner):
     def __init__(
-        self,
+        self, *args,
```

Comment:
If this `SupervisedLearner` is general purpose we can move it into app_common/pt/learners as Holger suggested.

Reply:
Agreed, I will also move the ditto learner to it since it is task agnostic.